api-lib/config.ts

```diff
@@ -31,3 +31,7 @@ export const LOCAL_SEED_ADDRESS = <string>(
   getEnvValue('LOCAL_SEED_ADDRESS', 'missing')
 );
 export const IS_LOCAL_ENV = process.env.NODE_ENV === 'development';
+
+export const COORDINAPE_USER_ADDRESS = <string>(
+    getEnvValue('COORDINAPE_USER_ADDRESS', '')
```

Comment:
dont add the '' because this should cause a fail if that isn't available

Reply:
updated